src/bpf_verifier.rs

```diff
@@ -0,0 +1,314 @@
+use rbpf::ebpf;
+
+// This “verifier” performs simple checks when the eBPF program is loaded into the VM (before it is
+// interpreted or JIT-compiled).
+
+fn verify_prog_len(prog: &[u8]) {
+    if prog.len() % ebpf::INSN_SIZE != 0 {
+        panic!(
```

Comment:
Can a Result<> be returned instead of panicking?

Reply:
This verifier was pretty much pulled directly from rbpf and contrary to it returning a bool rpbf ignores the return value and relies on panic.  This is another thing that's on my list of things to change about rbpf, fail gracefully